fpga/src/main/scala/arty100t/Configs.scala

```diff
@@ -45,22 +47,12 @@ class RocketArty100TConfig extends Config(
   new chipyard.config.WithBroadcastManager ++ // no l2
   new chipyard.RocketConfig)
 
-class UART230400RocketArty100TConfig extends Config(
-  new WithArty100TUARTTSI(uartBaudRate = 230400) ++
-  new RocketArty100TConfig)
-
-class UART460800RocketArty100TConfig extends Config(
-  new WithArty100TUARTTSI(uartBaudRate = 460800) ++
-  new RocketArty100TConfig)
-
-class UART921600RocketArty100TConfig extends Config(
-  new WithArty100TUARTTSI(uartBaudRate = 921600) ++
-  new RocketArty100TConfig)
-
-
 class NoCoresArty100TConfig extends Config(
   new WithArty100TTweaks ++
-  new chipyard.config.WithMemoryBusFrequency(50.0) ++
-  new chipyard.config.WithPeripheryBusFrequency(50.0) ++  // Match the sbus and pbus frequency
   new chipyard.config.WithBroadcastManager ++ // no l2
   new chipyard.NoCoresConfig)
+
+class BringupArty100TConfig extends Config(
+  new WithArty100TSerialTLToGPIO ++
```

Comment:
I think this won't work as is since the `ChipBringupHostConfig` has `WithSerialTLTiedOff`, is the `WithSerialTLTiedOff` necessary? Haven't tested this yet though.

Reply:
Or what happens if there are multiple harness binders for Serial TL?

Reply:
The WithARTY100TSerialTLToGPIO should override it. Does it not?

Reply:
I think I just don't understand how harness binders work fully, if they override one another then I think it should be fine. I haven't gotten a chance to test serial TL yet.

Reply:
Ok, I'll merge this for now then.